main/aspell/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "aspell"
+pkgver = "0.60.8.1"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gettext-devel",
+    "gmake",
+    "libtool",
+]
+pkgdesc = "Spell checker with good multi-language support"
+maintainer = "Isaac Freund <mail@isaacfreund.com>"
+license = "LGPL-2.1-only"
+url = "http://aspell.net"
+source = f"https://ftp.gnu.org/gnu/aspell/aspell-{pkgver}.tar.gz"
+sha256 = "d6da12b34d42d457fa604e435ad484a74b2effcd120ff40acd6bb3fb2887d21b"
+
+
+@subpackage("libaspell")
```

Comment:
should be called aspell-libs

Reply:
There's only one shared library in the package, `libaspell.so`. According to Packaging.md `libaspell` is correct: 

> However, if a library is a subpackage of a bigger software project, there are two things you can do. If the subpackage provides a single library that is usable as a standalone runtime dependency for other things, you should use the `lib` prefix.

I don't care strongly one way or the other, the docs should probably be updated if they're out of date though.

Reply:
ah indeed, I think my eyes glazed over the single letter difference or something. Done